agent/php_rinit.c

```diff
@@ -36,6 +36,7 @@ PHP_RINIT_FUNCTION(newrelic) {
   NRPRG(php_cur_stack_depth) = 0;
   NRPRG(deprecated_capture_request_parameters) = NRINI(capture_params);
   NRPRG(sapi_headers) = NULL;
+  NRPRG(error_group_user_callback).is_set = false;
```

Comment:
what about just set `NRPRG(error_group_user_callback) = {0};`

Reply:
does this ever need to be cleaned up in rshutdown? or it's pretty much there for the life of the agent?

Reply:
See this conversation for context on how/why this was implemented this way: https://github.com/newrelic/newrelic-php-agent/pull/719#discussion_r1307556367

Reply:
In general since this isn't a pointer there shouldn't be any cleanup required in rshutdown- resetting the `is_set` to false here in rinit allows us to set and unset it on a per-request basis without the need for allocating or deallocating memory.

Reply:
by cleanup, I meant resetting to unset so we won't be calling one application's callbacks on another application's errors.

Reply:
is that a risk if we set it to false at the beginning of every rinit?

Reply:
Nope, should be good.